src/lib/certmap/sss_certmap.c

```diff
@@ -180,11 +180,19 @@ static int parse_mapping_rule(struct sss_certmap_ctx *ctx,
     }
 
     if (type == NULL || strcmp(type, "LDAP") == 0) {
```

Comment:
Was this "LDAP:" prefix described before?
Current version of `man sss-certmap` (before this PR) doesn't seem to mention it?

Reply:
It was not needed, since there was only one version. And since it is the default if there is no other prefix it is not required in future, but I think it would be good to have a name for the default as well.